src/systems/a320_systems/src/pneumatic.rs

```diff
@@ -288,6 +288,22 @@ impl A320Pneumatic {
             .change_spatial_volume(yellow_hydraulic_reservoir.available_volume());
     }
 }
+impl PneumaticBleed for A320Pneumatic {
+    fn apu_bleed_is_on(&self) -> bool {
+        self.apu_bleed_air_valve.is_open()
+    }
+    fn engine_crossbleed_is_on(&self) -> bool {
+        self.cross_bleed_valve.is_open()
+    }
+}
+impl EngineStartState for A320Pneumatic {
+    fn left_engine_state(&self) -> EngineState {
+        self.fadec.engine_state(1)
+    }
+    fn right_engine_state(&self) -> EngineState {
+        self.fadec.engine_state(2)
+    }
+}
```

Comment:
I might have brought this upon myself, but I am not sure if we should be getting the engine state from the FADEC via the bleed system. Ideally, the FADEC should be independent (perhaps created in `A320` and passed into the pneumatic and the air conditioning system. This might not be within the scope of this PR though.

Reply:
I think you are right, we already have a `LeapEngine` module within `systems` that we will most likely have to extract into the `A320` specific area of the code at some point. Then it would be a good idea to get the FADEC out of `pneumatic` and merge it with the rest of the engine module. Might be a good idea for a small PR in the near future, maybe I have a look at it myself after this one.